src/helpers/parser/listItemParser.js

```diff
@@ -13,8 +13,10 @@ import { subtitle } from '../textHelper';
 
 const { ROOT_ROUTE_NAMES } = consts;
 
+const dataFilteredGenericTypes = [GenericType.Job, GenericType.Commercial];
```

Comment:
isn't this a constant and could be `DATA_FILTERED_GENERIC_TYPES`?
this would make `if (dataFilteredGenericTypes.includes(item?.genericType)) {` more readable in my opinion: `if (DATA_FILTERED_GENERIC_TYPES.includes(item?.genericType)) {`

Reply:
in addition, `GENERIC_TYPES` would be enough, right? to make the question "if generic types includes the items generic type"

Reply:
1. yes it could be a constant
2. it was supposed to be `date....` not `data....`, and with that in mind i don't like reducing the name to `GENERIC_TYPES`.

I will fix it once we agree on a naming :)

Reply:
the array contains two generic type strings. where do you see a date reference?

Reply:
because of the later context, right?
so what about `GENERIC_TYPES_WITH_DATES`?

Reply:
✅